mergify_engine/web/api/queues.py

```diff
@@ -36,6 +36,13 @@
 
 LOG = daiquiri.getLogger(__name__)
 
+router = fastapi.APIRouter(
+    tags=["queues"],
+    dependencies=[
+        fastapi.Depends(security.require_authentication),
+    ],
+)
+
```

Comment:
unrelated?

Reply:
It's related, I relax the default security dependencies of /v1 endpoint, I move them to queues/applications routers instead. So /v1/badges don't need authentication and this is reflexed in OpenAPI generated file.